com.visualstudio.code-oss.json

```diff
@@ -0,0 +1,180 @@
+                    "dest": ".electron"
+                },
+                {
+                    "type": "file",
+                    "url": "https://github.com/electron/electron/releases/download/v1.7.0/chromedriver-v1.7.0-linux-x64.zip",
+                    "sha512": "c2158bbd0c2ed28ad85cae9d400bd6416f0e4aa411e70287ccfaa4ad61b60692b17e0c6452a064ad55a396a5ad9e0253c5b3a0d75b91182995ab9fa25764d736",
+                    "dest": ".electron"
+                },
+                {
+                    "type": "file",
+                    "path": "SHASUMS256.txt-1.7.0",
+                    "dest": ".electron"
+                },
+                {
+                    "type": "file",
+                    "url": "https://github.com/electron/electron/releases/download/v1.7.7/electron-v1.7.7-linux-x64.zip",
+                    "sha512": "1c78c031442cddedab0923d474ccb4e12a21f3189b0f091db3df1a9c851f765a468e96bf4ee7cee2354af1237e5e26f12febd0c72945821f0c4e137733f86089",
+                    "dest": ".electron"
+                },
+                {
+                    "type": "file",
+                    "path": "SHASUMS256.txt-1.7.7",
+                    "dest": ".electron"
+                },
+                {
+                    "type": "file",
+                    "url": "https://github.com/electron/electron/releases/download/v1.7.9/electron-v1.7.9-linux-x64.zip",
+                    "sha512": "55c7c81f25632e95a1411da8465281d576da609c830e4126f2326ee272b89b8107d0e043ba334261fff6de66347ec3e570d5530f7ce85a348962d18e84843b98",
+                    "dest": "gulp-electron-cache/atom/electron"
+                },
+                {
+                    "type": "file",
+                    "url": "https://github.com/electron/electron/releases/download/v1.7.9/ffmpeg-v1.7.9-linux-x64.zip",
+                    "sha512": "5af1e2fcda46dc142f27252724041f295e16acf0884bacf8b8e8f5cb43d9c337f577eb0cce477830cc1c838ae6a4bb03ae8ac5eec4eba3505a9b741cea5feb7f",
+                    "dest": "gulp-electron-cache/atom/electron"
+                },
+                {
+                    "type": "file",
+                    "url": "https://github.com/roblourens/ripgrep/releases/download/0.7.1-patch.1/ripgrep-0.7.1-patch.1-linux-x64.zip",
+                    "sha512": "9ce9aac7198d2999c0c3ee9678c595139c6020c3529ac145502e1b44147694885dc6256aaad590c6a5cb349f751d8cad15e71bc6819619efc209758087dd024e",
+                    "dest": "misc"
+                },
+                "generated-sources.json",
+                {
+                    "type": "git",
+                    "url": "https://github.com/Microsoft/vscode.git",
+                    "tag": "1.20.1",
+                    "commit": "f88bbf9137d24d36d968ea6b2911786bfe103002",
+                    "dest": "vscode"
+                },
+                {
+                    "type": "file",
+                    "url": "https://upload.wikimedia.org/wikipedia/commons/2/2d/Visual_Studio_Code_1.18_icon.svg",
```

Comment:
There's an upstream icon for the OSS version and it's not this. If we're going to have two versions in flathub they can't be confusing.

Reply:
OK, I will change